src/test/java/me/botsko/prism/utils/UtilitiesTest.java

```diff
@@ -0,0 +1,35 @@
+package me.botsko.prism.utils;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import me.botsko.prism.utils.block.Utilities;
+import org.bukkit.Material;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created for use for the Add5tar MC Minecraft server
+ * Created by benjamincharlton on 23/05/2020.
+ */
+public class UtilitiesTest {
+
+    private ServerMock server;
```

Comment:
Codacy found an issue: [Perhaps 'server' could be replaced by a local variable.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5612535)